### PR TITLE
[ES] Update basics.md

### PR DESCRIPTION
**Pull request by guty**
A couple of translation mistakes.

### CODE REVIEW

File: es/lessons/basics/basics.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ iex> 0x1F
 
 ### Coma flotante
 
-En Elixir, los números con coma flotante requieren un decimal después de al menos un dígito; estos tienen una precisión de 64 bits y soportan `e` para números exponenciales.
+En Elixir, los números con coma flotante requieren un decimal antes de al menos un dígito; estos tienen una precisión de 64 bits y soportan `e` para números exponenciales.
```
Comment (erickgnavar) [inline es/lessons/basics/basics.md]: ¿a qué se debe este cambio? el texto original en inglés es:
> floating point numbers require a decimal **after** at least one digit

**Comment by doomspork** [comment]
Thank you and welcome @guty!

**Comment by doomspork** [comment]
@guty and @erickgnavar is this good to merge?

**Comment by erickgnavar** [comment]
> @guty and @erickgnavar is this good to merge?

Nope, a small change is required yet

**Comment by doomspork** [comment]
@guty interested in getting this change in and merged for Hacktoberfest?

**Comment by doomspork** [comment]
@erickgnavar would you be up for adding a `suggestion` to the PR with your proposed change so we can apply it and merge?

File: es/lessons/basics/basics.md (hunk 1 above)
Comment (erickgnavar) [reply es/lessons/basics/basics.md]:
```suggestion
En Elixir, los números con coma flotante requieren un decimal después de al menos un dígito; estos tienen una precisión de 64 bits y soportan `e` para números exponenciales.
```

**Comment by erickgnavar** [comment]
> @erickgnavar would you be up for adding a `suggestion` to the PR with your proposed change so we can apply it and merge?

Sure, I did it already

File: es/lessons/basics/basics.md (hunk 1 above)
Comment (doomspork) [reply es/lessons/basics/basics.md]: That did not go as expected 😞 
image

I will merge this and push a commit with your change.

**Pull request merged**